.github/workflows/daily.yml

```diff
@@ -89,6 +89,14 @@ jobs:
         include:
           - device: "iPhone 13"
             os_version: "15"
+          - device: "iPhone 14"
+            os_version: "16"
+          - device: "iPhone 15"
+            os_version: "17"
+          - device: "Xiaomi Redmi Note 11"
```

Comment:
just curious, but is there anything special abouto the xiami note?

Reply:
no, I picked a Xiaomi because it has the second biggest Android market share after Samsung